pipelines/rj_crm__disparo_template/utils/dispatch.py

```
@@ -1295,10 +1295,28 @@ def send_to_sftp(
     sftp_password: str = None,
     sftp_port: int = 22,
     sftp_remote_path: str = "/",
+    sftp_host_key: str = None,
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)